src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="main-container">
```

Comment:
Please use 'secondary-global-button' and 'primary-global-button' styles

Reply:
fixed